stylesheets/login.css

```diff
@@ -0,0 +1,49 @@
+.login-container {
+	width: 370px;
```

Comment:
use rem as units

Reply:
alright harshit ! thanks , will update that in next commit